src/sentry/static/sentry/app/views/projectInstall/platform.jsx

```diff
@@ -126,14 +124,38 @@ class ProjectInstallPlatform extends React.Component {
           )}
 
           {this.isGettingStarted && (
-            <Button
-              priority="primary"
-              size="large"
-              to={issueStreamLink}
-              style={{marginTop: 20}}
+            <Projects
+              key={`${orgId}-${projectId}`}
+              orgId={orgId}
+              slugs={[projectId]}
+              passthroughPlaceholderProject={false}
             >
-              {t('Got it! Take me to the Issue Stream.')}
-            </Button>
+              {({projects, initiallyLoaded, fetching, fetchError}) => {
+                const projectsLoading = !initiallyLoaded && fetching;
+                const issueStreamLinkQuery =
+                  !projectsLoading && !fetchError && projects.length
+                    ? {
+                        project: projects[0].id,
+                      }
+                    : {};
+
+                return (
+                  <Button
+                    priority="primary"
+                    size="large"
+                    busy={projectsLoading}
+                    to={{
+                      pathname: issueStreamLink,
+                      query: issueStreamLinkQuery,
+                      hash: '#welcome',
+                    }}
+                    style={{marginTop: 20}}
+                  >
+                    {t('Got it! Take me to the Issue Stream.')}
+                  </Button>
```

Comment:
I opted to not include a loading indicator here as this request should load fast enough vs.  the user having to scroll through page and click. In the worst case, they still get taken to issues stream.